### DISCUSSION TITLE
LuaRocks Installation in Win 10 does nothing

### DISCUSSION THREAD

**Discussion by albertleng** [keplerproject/luarocks#1283] [Q&A]
I just started learning Lua few days ago. 

My system:
```
Windows 10
Lua 5.2.4  Copyright (C) 1994-2015 Lua.org, PUC-Rio
```

I downloaded and unzipped [LuaRocks-3.5.0-Windows-64.zip](http://luarocks.github.io/luarocks/releases/luarocks-3.5.0-windows-64.zip). After clicking the exe and allowing the running permission, I could see a flickering command prompt. It just disappeared.

What shall I do next to install it?

Thanks.

**Comment by daurnimator** [comment]
Open a windows command prompt, navigate to the directory where you unzippe it, and run `luarocks.exe`

**Comment by C3pa** [comment]
I've written about my journey on using luarocks on Windows in #1501.
